framework_tvb/tvb/adapters/visualizers/topographic.py

```diff
@@ -277,6 +277,11 @@ def launch(self, view_model):
         for i, array_data in enumerate(arrays):
             try:
                 data_array = TopographyCalculations.compute_topography_data(array_data, sensor_locations)
+                first_label = h5.load_from_index(connectivities_idx[0]).region_labels[0]
```

Comment:
Isn't it better to use connectivity.hemisphere array of Booleans instead of hard coding the Letter R?

Reply:
How about this improvement?